src/components/ArticleCard.jsx

```diff
@@ -7,23 +7,31 @@ export default function ArticleCard(props) {
   const topic = articleCard.topic;
   const article_img_url = articleCard.article_img_url;
   const votes = articleCard.votes;
-  const date = articleCard.created_at;
+  const created_at = articleCard.created_at;
   const article_id = articleCard.article_id;
 
+  const dateObject = new Date(created_at);
```

Comment:
As a rule of thumb adding words like `object` `array` `string` etc to variables isn't necessary